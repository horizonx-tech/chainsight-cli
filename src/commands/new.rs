use clap::Parser;

#[derive(Debug, Parser)]
#[command(name = "new")]
/// Generate ChainSight project by prepared template
pub struct NewOpts {}