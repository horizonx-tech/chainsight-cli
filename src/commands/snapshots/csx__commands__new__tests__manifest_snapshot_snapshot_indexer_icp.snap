---
source: src/commands/new.rs
expression: template_snapshot_indexer_icp_manifest(&id).to_str_as_yaml().unwrap()
---
# yaml-language-server: $schema=https://raw.githubusercontent.com/horizonx-tech/chainsight-cli/main/resources/schema/snapshot_indexer_icp.json
version: v1
metadata:
  label: Sample Snapshot Indexer Icp
  type: snapshot_indexer_icp
  description: ''
  tags:
  - ERC-20
  - Ethereum
  - DAI
datasource:
  location:
    id: sample_snapshot_indexer_evm
  method:
    identifier: 'get_last_snapshot : () -> (record { value : text; timestamp : nat64 })'
    interface: null
    args: []
storage:
  with_timestamp: true
interval: 3600
lens_targets: null

